cli/src/cmd_auth.rs

```diff
@@ -124,20 +124,26 @@ pub fn parse_host(input: &str) -> Result<url::Url> {
 ///     $ oxide auth login --host http://oxide.internal
 #[derive(Parser, Debug, Clone)]
 #[clap(verbatim_doc_comment)]
+#[command(group(ArgGroup::new("browser-opt").args(["browser", "no_browser"])))]
 pub struct CmdAuthLogin {
     /// Read token from standard input.
     #[clap(long)]
-    pub with_token: bool,
+    with_token: bool,
 
     /// The host of the Oxide instance to authenticate with.
     /// This assumes http; specify an `http://` prefix if needed.
     #[clap(short = 'H', long, env = "OXIDE_HOST", value_parser = parse_host)]
-    pub host: url::Url,
+    host: url::Url,
+
+    /// Override the default browser when opening the authentication URL.
+    #[clap(long)]
```

Comment:
User gets a nice error if they attempt to use both flags :)

```suggestion
    #[clap(long, conflicts_with = "no-browser")]
```

Reply:
it was already working that way, albeit in a weird way that I had copied from a `typify` community contribution. Made it more normal.